### PR TITLE
adding cpp demo notebook

### PR DESCRIPTION
**Pull request by vinhngx**

### CODE REVIEW

File: tftrt/examples/cpp/image-classification/README.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
+```
```
Comment (pooyadavoodi) [inline tftrt/examples/cpp/image-classification/README.md]: Could you remove the license from the readme?

Comment (pooyadavoodi) [reply tftrt/examples/cpp/image-classification/README.md]: And please add the license to other files.

Comment (vinhngx) [reply tftrt/examples/cpp/image-classification/README.md]: I removed copyright in README added copyrights to other files. Pls note some files are built upon Google code, thus double copyright. Pls review. Thanks.

**Comment by pooyadavoodi** [comment]
Thanks @vinhngx

**Pull request merged**